src/OpenLoco/src/Ui/ScrollFlags.hpp

```diff
@@ -0,0 +1,59 @@
+
```

Comment:
Could you add a pragma once. Also try ensure this is standalone. `#include <OpenLoco/Core/EnumFlags.hpp>` Also add the header to the headers section of the CMakeLists.txt (then it will kick in the standalone checks which will tell you you are missing EnumFlags)